work/flow.rst

```diff
@@ -284,7 +282,7 @@ Before offering unsolicited code review on your colleagues' user branches, remem
 Developers can feel free to rebase and force push work to their personal user branches.
 
 A user branch *cannot* be merged into main; it must be converted into a *ticket branch* first.
-The only exception is for documentation edits and additions.
+The only exception is for documentation fixes.
```

Comment:
I'm not sure that everyone would interpret "fixes" to include "additions".

Reply:
My assumption was that additions don't count, because they can break things. Maybe this, which I think captures the intent?

```
The only exception is for documentation fixes in packages, or edits and additions to this developer guide.
```

Reply:
The original intent was to try to make contributing to the documentation as low-cost as possible, even removing the slight friction of creating a Jira ticket describing what documentation was to be added or modified.  That includes docstrings, package doc pages, or guides.

Even PRs from `u` branches still get GHA checks and merge protections.

Reply:
Thanks for clarifying: I've made use of that. How about this version?

https://github.com/lsst-dm/dm_dev_guide/pull/535/commits/34ab153a94f69d1d2818d199f81be0b275ac98b0

Reply:
Looks great.